server/public/styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  background-color: rgb(240, 240, 240);
+}
+
+.title {
+  background-color: #343434;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+
+}
```

Comment:
Bootstrap has text colors too

https://getbootstrap.com/docs/4.4/utilities/colors/#color

and I think the font family is already set to Roboto or something with bootstrap.